main.js

```diff
@@ -176,22 +176,38 @@ define([
                         return;
                     }
 
-                    if (mapLayer instanceof TileLayer || mapLayer instanceof MapImageLayer) {
+                    if (mapLayer instanceof TileLayer) {
                         if (layerServiceIds.length === 0) {
                             mapLayer.visible = false;
                         } else {
                             mapLayer.visible = true;
                         }
                     } else {
-                        // `.setVisibleLayers` seems to be unavailable in
```

Comment:
Good call. I'm not sure. I'll update it to only apply to `MapImageLayer` objects.